### PR TITLE
add unit test coverage for pkg/util/labels

### PR DESCRIPTION
**Pull request by zhoumingcheng**
Signed-off-by: zhoumingcheng <zhoumingcheng@beyondcent.com>

#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Improve test coverage

Run test coverage command:
```
go test -cover ./pkg/util/labels/
```
Befor:
```
ok      k8s.io/kubernetes/pkg/util/labels/     0.019s  coverage: 78.7% of statements
```
After:
```
ok      k8s.io/kubernetes/pkg/util/labels/    0.008s  coverage: 100.0% of statements
```
#### Which issue(s) this PR fixes:

Fixes #

#### Special notes for your reviewer:
Thank you！
#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/util/labels/labels_test.go [hunk 1]
```diff
@@ -148,11 +149,47 @@ func TestCloneSelectorAndAddLabel(t *testing.T) {
...
 	}
 
 	for _, tc := range cases {
-		ls_in := metav1.LabelSelector{MatchLabels: tc.labels}
-		ls_out := metav1.LabelSelector{MatchLabels: tc.want}
+		ls_in := metav1.LabelSelector{MatchLabels: tc.labels, MatchExpressions: tc.matchExpressions}
+		ls_out := metav1.LabelSelector{MatchLabels: tc.want, MatchExpressions: tc.matchExpressions}
 
 		got := CloneSelectorAndAddLabel(&ls_in, tc.labelKey, tc.labelValue)
 		if !reflect.DeepEqual(got, &ls_out) {
```
Comment (kerthcet) [inline pkg/util/labels/labels_test.go:195]: It's not right here
```
t.Errorf("got %v, want %v", got, tc.want)
```
tc.want is only part of `CloneSelectorAndAddLabel`'s result.

**Comment by zhoumingcheng** [comment]
@kerthcet 
Hello, I would like to consult this error when testing https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/110852/pull-kubernetes-verify/1542321068105011200. There is a .readonly file in this directory, is it that the content in this directory cannot be changed?

**Comment by kerthcet** [comment]
> @kerthcet Hello, I would like to consult this error when testing https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/110852/pull-kubernetes-verify/1542321068105011200. There is a .readonly file in this directory, is it that the content in this directory cannot be changed?

You can run `hack/verify-shellcheck.sh` locally to verify this.

**Comment by kerthcet** [comment]
As the name indicates, I think it's read only.

**Comment by zhoumingcheng** [comment]
> As the name indicates, I think it's read only.

thanks, i will close this pr.

**Comment by kerthcet** [comment]
Just try to solve your confusions why we readonly here: anything that uses this code should be using meta/v1 code instead. You can refer to https://github.com/kubernetes/kubernetes/pull/80685 for more discussions. 

As a follow up, I will raise an issue to discuss about remove the folder totally. Thanks for your contributions anyway.

**Comment by kerthcet** [comment]
Package label is still in use by other places, so we can't delete them at this moment.

**Pull request closed without merging**